core/templates/dev/head/domain/utilities/UrlInterpolationService.ts

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 require('services/AlertsService.ts');
+require('services/contextual/UrlService.ts');
```

Comment:
I meant even after UtilsService require. We first order by how deep is the file in the folder structure and then by alphabetical order.

Reply:
Ohh okay

Reply:
Not done?